### PR TITLE
Add regression benchmarks to PMLB release

### PR DESCRIPTION
**Pull request by rhiever**

### CODE REVIEW

**Comment by rhiever** [comment]
Looks like there's some merge conflicts with a couple of the datasets, @weixuanfu2016. Is it because they're named `.csv.gz` and not `.tsv.gz`?

**Comment by weixuanfu** [comment]
@rhiever hmm, these two files should be deleted and all reformatted ones are named with `tsv.gz`. Check links below:

https://github.com/EpistasisLab/penn-ml-benchmarks/tree/development/datasets/classification/churn

https://github.com/EpistasisLab/penn-ml-benchmarks/tree/development/datasets/classification/threeOf9

File: datasets/regression/Regression_datasets_pmlb.tsv [hunk 1]
```diff
@@ -0,0 +1,121 @@
+did	MajorityClassSize	MaxNominalAttDistinctValues	MinorityClassSize	NumberOfClasses	NumberOfFeatures	NumberOfInstances	NumberOfInstancesWithMissingValues	NumberOfMissingValues	NumberOfNumericFeatures	NumberOfSymbolicFeatures	name	status
```
Comment (weixuanfu) [inline datasets/regression/Regression_datasets_pmlb.tsv:1]: We may also need a meta feature table for classification datasets.

**Comment by rhiever** [comment]
With the merge of #11, I think this PR needs to be rebased. May be easier to start from scratch?

**Comment by weixuanfu** [comment]
I think so. Formatting or adding datasets may cause tons of conflicts.

**Comment by rhiever** [comment]
I think it's just a matter of moving the datasets around a bit now that @lacava / @athril's PR is merged.

**Comment by rhiever** [comment]
I'll go ahead and close this PR.

**Comment by lacava** [comment]
once datasets are added, you should just be able to run pmlb/write_metadata.py to generate the readme files.

**Pull request closed without merging**